### ISSUE TITLE
JA: Multimonitor handling is broken on Linux

### ISSUE THREAD

**Issue by Civil** [JACoders/OpenJK#783]
Mouse stops working (I can see OS's cursor, but ingame one doesn't move) when playing fullscreen with multiple monitors on Linux.

**Comment by ensiform** [comment]
Please verify your issue as being OpenJK first, as it is SDL problem.

**Comment by xycaleth** [comment]
How do you know it's an SDL problem?

**Comment by ensiform** [comment]
Because multimon support was known to be broken still on Linux for SDL ._.

**Comment by Civil** [comment]
I disagree with that, because I never saw this issue with other SDL-based games under Linux. Mostlikely they are doing something differently, like instead of Fullscreen having fake-fullscreen under linux. So it's still *can* and *must* be worked around by you.

**Comment by ensiform** [comment]
We will NOT be making OS specific workarounds. _THAT_ is the must.

And I disagree because you're comparing apples to oranges with older SDL1 games etc.

**Comment by Civil** [comment]
I'm comparing with current Steam games, which ALL are using SDL2, for example Wasteland 2.

If you don't have developers now, maybe you'll just leave this case open until there'll be somebody to work on this particular issue?

As about Configuration details - it's all on my laptop:
Gentoo, kernel 4.2.3, KDE 5.5 (kwin as windowmanager), i7-4710HQ, Intel HD4600, xf86-video-intel 2.99.917, xorg-server 1.17.4, SDL 2.0.3.

Is there anything else you need to accept this bug report?

**Comment by ensiform** [comment]
Are you explicitly using both monitors for the game, a specific monitor, is it not primary etc.

**Comment by Civil** [comment]
Sources for wasteland are not available. But if you know some open-source games that uses SDL2 you can name it and I'll also try how it works there.

Yeah, all r_fullscreen 0 are working ok.

About primary/secondary - it was running on secondary display, but it doesn't matter - cause running on primary causes the same behavior.

**Comment by Civil** [comment]
Sources for wasteland are not available. But if you know some open-source games that uses SDL2 you can name it and I'll also try how it works there.

Yeah, all r_fullscreen 0 are working ok.

About primary/secondary - it was running on secondary display, but it doesn't matter - cause running on primary causes the same behavior.

**Comment by ensiform** [comment]
ioquake3 test builds use the same implementation we do.  ETLegacy has some changes but mostly same.  Don't know how different unvanquished is.  Unfortunately the latter two are governed under GPLv3 and we're not 2+ and can't change anyway.

**Comment by Civil** [comment]
I'll try to test it more in next couple of weeks, when I'll have more time.

**Comment by ensiform** [comment]
Yes some do, but we won't be doing that because it is intertwined unless its simple enough.

**Comment by ensiform** [comment]
It is kind of sad that there is no good implementation out there for people to use under various licenses.
